src/doc/reference.md

```diff
@@ -3888,7 +3883,7 @@ Box<i32>, y: Box<i32>)` declare one mutable variable `x` and one immutable
 variable `y`).
 
 Methods that take either `self` or `Box<Self>` can optionally place them in a
-mutable slot by prefixing them with `mut` (similar to regular arguments):
+mutable variable by prefixing them with `mut` (similar to regular arguments):
```

Comment:
Sure, binding is fine. Slot is just very old terminology, the only uses in this sense remaining are weird old corners of trans inside the compiler. In the docs it's a dead term, sticks out.